android/src/main/java/com/swmansion/rnscreens/SearchBarView.kt

```diff
@@ -144,6 +150,14 @@ class SearchBarView(reactContext: ReactContext?) : ReactViewGroup(reactContext)
         text?.let { screenStackFragment?.searchView?.setText(it) }
     }
 
+    private fun setToolbarElementsVisibility(visibility: Int) {
+        for (i in 0..(headerConfig?.configSubviewsCount?.minus(1) ?: 0)) {
```

Comment:
Just a one note about this line - since we need to iterate over the subviews I need to have closed range on the end of the subviews array. That's why I'm decrementing the count by one, but we can make this line better with the newly-introduced `rangeUntil` method for the ranges:

```suggestion
        val subviewsCount = headerConfig?.configSubviewsCount? ?: 0
        for (i in 0..<subviewsCount) {
```

which won't count the last element of a range. Unfortunately this requires us to update Kotlin to 1.9.10 - that's why I've made a PR #1904 to update Kotlin version.

Reply:
I believe that using any recent Kotlin syntax would force users to manually bump Kotlin versions in their applications build scripts. We should not do that.

Reply:
@kkafar Sure!